packages/components/psammead-podcast-promo/src/components/card-link.jsx

```diff
@@ -0,0 +1,29 @@
+import styled from '@emotion/styled';
+
+import { C_PEBBLE } from '@bbc/psammead-styles/colours';
+
+const CardLink = styled.a`
+  color: unset;
```

Comment:
I think this `a` needs to have its default underline removed.

```suggestion
  color: unset;
  text-decoration: none;
```